ocs_ci/ocs/resources/mcg.py

```
@@ -861,26 +861,27 @@ def _get_mirroring_percentage():
+                    logger.info("All objects mirrored successfully.")
+                    return
+        except TimeoutExpiredError:
+            observed = (
+                f"{last_mirror_percentage}%"
+                if last_mirror_percentage is not None
+                else "unknown"
+            )
+            logger.error(
+                f"Mirroring did not complete within {timeout}s. "
+                f"Last percentage: {observed}"
+            )
```

Ruff (0.15.13)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)